specification/keyvault/resource-manager/Microsoft.KeyVault/stable/2022-11-01/keys.json

```diff
@@ -0,0 +1,671 @@
+        "created": {
+          "type": "integer",
+          "format": "int64",
+          "readOnly": true,
+          "description": "Creation time in seconds since 1970-01-01T00:00:00Z."
+        },
+        "updated": {
+          "type": "integer",
+          "format": "int64",
+          "readOnly": true,
+          "description": "Last updated time in seconds since 1970-01-01T00:00:00Z."
+        },
+        "expiryTime": {
+          "type": "string",
+          "description": "The expiration time for the new key version. It should be in ISO8601 format. Eg: 'P90D', 'P1Y'."
+        }
+      },
+      "type": "object"
+    },
+    "LifetimeAction": {
+      "properties": {
+        "trigger": {
+          "$ref": "#/definitions/Trigger",
+          "description": "The trigger of key rotation policy lifetimeAction."
+        },
+        "action": {
+          "$ref": "#/definitions/Action",
+          "description": "The action of key rotation policy lifetimeAction."
+        }
+      },
+      "type": "object"
+    },
+    "Trigger": {
+      "properties": {
+        "timeAfterCreate": {
+          "type": "string",
+          "description": "The time duration after key creation to rotate the key. It only applies to rotate. It will be in ISO 8601 duration format. Eg: 'P90D', 'P1Y'."
+        },
+        "timeBeforeExpiry": {
+          "type": "string",
+          "description": "The time duration before key expiring to rotate or notify. It will be in ISO 8601 duration format. Eg: 'P90D', 'P1Y'."
+        }
+      },
+      "type": "object"
+    },
+    "Action": {
+      "properties": {
+        "type": {
+          "type": "string",
+          "description": "The type of action.",
+          "enum": [
+            "rotate",
+            "notify"
+          ],
+          "x-ms-enum": {
+            "name": "KeyRotationPolicyActionType",
+            "modelAsString": false
+          }
+        }
+      },
+      "type": "object"
+    },
+    "KeyReleasePolicy": {
+      "properties": {
+        "contentType": {
+          "description": "Content type and version of key release policy",
+          "type": "string",
+          "default": "application/json; charset=utf-8"
+        },
+        "data": {
+          "description": "Blob encoding the policy rules under which the key can be released.",
+          "type": "string",
+          "format": "base64url"
+        }
+      },
+      "type": "object"
+    }
+  },
+  "parameters": {
+    "SubscriptionIdParameter": {
+      "name": "subscriptionId",
+      "in": "path",
+      "required": true,
+      "type": "string",
+      "description": "Subscription credentials which uniquely identify Microsoft Azure subscription. The subscription ID forms part of the URI for every service call."
+    },
+    "ApiVersionParameter": {
+      "name": "api-version",
+      "in": "query",
+      "required": true,
+      "type": "string",
+      "description": "Client Api Version."
+    }
+  },
```

Comment:
You are highly recommended to remove your own definitions of these and use the definitions from common-specification/types/v3/types.json instead

Reply:
this is old existing file, not introduced in this PR, so skipping.